stories/assets/js/Carousel.js

```diff
@@ -2,47 +2,47 @@ import arrowright from '../images/Icon/arrow-right.svg';
 import arrowleft from '../images/Icon/arrow-left.svg';
 
 export function CarouselFun(ele, gapele, gapele_sm, viewcard) {
-
   /**
-   * A Custom component to get Glide Slides Length and Width.
+   * A Custom Length component to get Glide Slides Length.
```

Comment:
@neelesh-tippe Kindly verify the file as per develop branch. Please don't revert the changes from develop.

Reply:
done